.circleci/config.yml

```diff
@@ -0,0 +1,380 @@
+          no_output_timeout: 10m
+      - run:
+          command: yarn grunt test:jquery-2.1 --browsers="$BROWSERS" --reporters=spec
+          no_output_timeout: 10m
+
+  e2e-test-1:
+    executor:
+      name: default-executor
+    steps:
+      - run_e2e_tests:
+          specs: test/e2e/tests/**/*.js
+
+  e2e-test-2a:
+    executor:
+      name: default-executor
+    steps:
+      - run_e2e_tests:
+          specs: build/docs/ptore2e/example-ng*/**/default_test.js
+
+  e2e-test-2b:
+    executor:
+      name: default-executor
+    steps:
+      - run_e2e_tests:
+          specs: "build/docs/ptore2e/!(example-ng*)/**/default_test.js"
+
+  e2e-test-jquery-1:
+    executor:
+      name: default-executor
+    steps:
+      - run_e2e_tests_jquery:
+          specs: test/e2e/tests/**/*.js
+
+  e2e-test-jquery-2a:
+    executor:
+      name: default-executor
+    steps:
+      - run_e2e_tests_jquery:
+          specs: build/docs/ptore2e/example-ng*/**/jquery_test.js
+
+  e2e-test-jquery-2b:
+    executor:
+      name: default-executor
+    steps:
+      - run_e2e_tests_jquery:
+          specs:  build/docs/ptore2e/!(example-ng*)/**/jquery_test.js
+
+  deploy-docs:
+    executor:
+      name: default-executor
+    steps:
+      - custom_attach_workspace
+      - init_environment
+      - run: yarn grunt prepareDeploy
+        # Install dependencies for Firebase functions to prevent parsing errors during deployment
+        # See https://github.com/angular/angular.js/pull/16453
+      - run: yarn -cwd $PROJECT_ROOT/scripts/docs.angularjs.org-firebase/functions
+      - run: yarn firebase deploy --token "$FIREBASE_TOKEN" --only hosting
+
+  deploy-code:
+    executor:
+      name: default-executor
+    steps:
+      - run:
+          name: Authenticate and configure Docker
+          command: |
+            echo $GCLOUD_SERVICE_KEY | gcloud auth activate-service-account --key-file=-
+            gcloud --quiet config set project ${GOOGLE_PROJECT_ID}
+      - run:
+          name: Sync files to code.angularjs.org
+          command: |
+            gsutil -m rsync $PROJECT_ROOT/deploy/code gs://code-angularjs-org-338b8.appspot.com
+
+workflows:
+  version: 2
+  default_workflow:
+    jobs:
+      - setup
+      - lint:
+          requires:
+            - setup
+      - unit-test:
+          requires:
+            - setup
+      - unit-test-jquery:
+          requires:
+            - setup
+      - e2e-test-1:
+          requires:
+            - setup
+      - e2e-test-2a:
+          requires:
+            - setup
+      - e2e-test-2b:
+          requires:
+            - setup
+      - e2e-test-jquery-1:
+          requires:
+            - setup
+      - e2e-test-jquery-2a:
+          requires:
+            - setup
+      - e2e-test-jquery-2b:
+          requires:
+            - setup
+      - deploy-docs:
+          filters:
+            branches:
+              only:
+                - latest
```

Comment:
Is `latest` a special value? Are there any docs on how it works?

Reply:
Previously deployments of the "latest" build, i.e. the releases that were tagged with a `latest` npm dist-tag also triggered deployment of the docs to a firebase hosting, and also a special version of the code to `snapshot-stable`.

This is more difficult in CircleCI, so I had this idea that we would simply keep a new `latest` git branch updated to the latest stable release, which would then trigger the deployments.
I never got it fleshed out properly - and now I am thinking, given the status of the project, that we can just run manual deployments when we run the release. WDYT?

Reply:
Why is it more difficult in CircleCI. It seems quite straight forward to me (just check `package.json > distTag` and abort the job if it is not `latest`).

Reply:
Hmm, I guess that could work. I was thinking of ways to avoid running the job in the first place.